src/dune

```diff
@@ -2,4 +2,4 @@
  (public_name index)
  (name index)
  (modules_without_implementation io)
- (libraries logs fmt))
+ (libraries logs logs.threaded fmt threads))
```

Comment:
is there any hope to expose the right hooks in `IO` and not depend directly on threads in the core library?